app/views/admin/kaminari/_gap.html.haml

```diff
@@ -1,2 +1,2 @@
-%li.disabled.inline-item.align-bottom
-  = content_tag :a, raw(t 'views.pagination.truncate')
+-# %li.disabled.list-inline-item.align-bottom.pt-2
```

Comment:
I don't think you should comment these things out. I think you should remove them and remove the places that render them.